Library/Formula/sile.rb

```diff
@@ -0,0 +1,37 @@
+class Sile < Formula
+  desc "Modern typesetting system inspired by TeX"
+  homepage "http://www.sile-typesetter.org/"
+  url "https://github.com/simoncozens/sile/releases/download/v0.9.2/sile-0.9.2.tar.bz2"
+  sha256 "2223582818df06daa4609cee40a81e8787085ad1795d4b2ce5edbe0663b74e18"
+
+  head do
+    url "https://github.com/simoncozens/sile.git"
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "harfbuzz" => :build
+  depends_on "fontconfig" => :build
+  depends_on "libpng" => :build
+  depends_on "freetype" => :build
+  depends_on "lua"
+
+  def install
+    system "luarocks-5.2", "install", "lpeg"
+    system "luarocks-5.2", "install", "luaexpat"
+    system "./bootstrap.sh" if build.head?
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--with-lua=#{prefix}",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "sile", "--version"
```

Comment:
Can we have a test that actually does something and e.g. write some input with `Pathname.write` and pass that in? Thanks!